source/includes/indexes/indexes.kt

```diff
@@ -51,4 +53,33 @@ fun main() {
         println(result)
     }
     // end-index-single-query
+
+    // start-create-search-index
+    val index = Document("mappings", Document("dynamic", true))
```

Comment:
Do we explain what this line of code is doing anywhere? I understand that it's index options, but I'm not really sure what it means specifically. I think it would be helpful to either explain it on this page somewhere, or link to somewhere that does.

Reply:
Can add a link out to Atlas to address this